comp/forwarder/orchestrator/component.go

```diff
@@ -0,0 +1,20 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+// Package orchestrator implements the orchestrator forwarder component.
+package orchestrator
+
+import "github.com/DataDog/datadog-agent/comp/forwarder/defaultforwarder"
+
+// team: agent-metrics-logs
+
+// Component is the component type.
+type Component interface {
+	Get() (defaultforwarder.Forwarder, bool)
+
+	// TODO: (components): This function is used to know if Stop was already called.
+	// Remove it when Stop is not part of this interface.
+	Reset()
```

Comment:
Is the comment up-to-date since the `Reset` method doesn't return anything ?

Reply:
Reset makes `get` method returns `false` which is used to know if `Stop` must be called. I will update the comment to reflect that.